modules/analyzing-the-collector-pod-status.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * troubleshooting/retrieving-and-analyzing-the-collector-logs-and-pod-status.adoc
+:_module-type: PROCEDURE
+[id="analyzing-the-collector-pod-status_{context}"]
+= Analyzing the Collector pod status
+
+Examining the pod’s most recent status is another easy way to determine the cause of a Collector crash. Failure messages are recorded to the most recent status and are accessible using the `kubectl` or `oc` command:
```

Comment:
**issue:** This is a Procedure module, its missing the `. Procedure` thingy. The steps should be a list. Also example output should come under the command. PS the following examples:
- https://docs.openshift.com/container-platform/4.12/logging/troubleshooting/cluster-logging-cluster-status.html
- https://docs.openshift.com/container-platform/4.12/backup_and_restore/control_plane_backup_and_restore/replacing-unhealthy-etcd-member.html#restore-determine-state-etcd-member_replacing-unhealthy-etcd-member

Reply:
ack